src/libraries/System.Net.Requests/src/System/Net/FtpWebRequest.cs

```diff
@@ -958,9 +958,9 @@ private async void CreateConnectionAsync()
             object result;
             try
             {
-                var client = new TcpClient();
+                var client = new Socket(SocketType.Stream, ProtocolType.Tcp);
```

Comment:
In https://github.com/dotnet/runtime/issues/67879 you pointed out differences in `TcpClient` connect methods. Will this change keep the same behavior as the previous code?
If not, do we actually care?

Reply:
I was primarily worried about the difference between IPv4 and dual-mode socket. Since this uses `ConnectAsync` it would fall-back straight to `Socket.ConnectAsync` e.g. the is no functional change,